src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandlerTest.java

```diff
@@ -729,8 +728,11 @@ void shouldReturnExpectedSubmittedCallbackResponse_whenRespondent1SolicitorNotRe
 
                 assertThat(response).usingRecursiveComparison().isEqualTo(
                     SubmittedCallbackResponse.builder()
-                        .confirmationHeader(format("# Your claim will now progress offline: %s", REFERENCE_NUMBER))
-                        .confirmationBody(UNREGISTERED_ORG_CONFIRMATION_BODY)
+                        .confirmationHeader(format("# Your claim has been received and will progress offline%n## "
+                                                       + "Claim number: %s", REFERENCE_NUMBER))
+                        .confirmationBody(format(LIP_CONFIRMATION_BODY,
```

Comment:
I think this is a really bad way to test this, no way we can see if it's accidentally been changed